pontoon/sync/core.py

```diff
@@ -165,6 +165,26 @@ def update_translated_resources(db_project, vcs_project, locale):
                 translatedresource.calculate_stats()
 
 
+def update_translated_resources_no_files(db_project, locale, changed_resources):
```

Comment:
This is the main change. It updates translated resources for all sources that have source string changed.